<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMainMenuGUI</class>
 <widget class="QDialog" name="frmMainMenuGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>easy fMRI (2.0)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icon.qrc">
    <normaloff>./icons/icons8-brain-128.png</normaloff>./icons/icons8-brain-128.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>14</y>
     <width>611</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>easy fMRI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>701</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QToolButton" name="btnPreprocess">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Preprocessing (FSL)</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/402612-200.png</normaloff>./icons/402612-200.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnFeatureAnalysis">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Feature Analysis</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/icons8-scatter-plot-filled-64.png</normaloff>./icons/icons8-scatter-plot-filled-64.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnModelAnalysis">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Model Analysis</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/noun_mind_1705433.png</normaloff>./icons/noun_mind_1705433.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnVisualization">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualization (AFNI)</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/noun_Brain_7784.png</normaloff>./icons/noun_Brain_7784.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tools</string>
    </attribute>
    <widget class="QComboBox" name="cbTools">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>491</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select a tool:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnTools">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/icons8-play-100.png</normaloff>./icons/icons8-play-100.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Mode</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Easy fMRI DIR (set from $EASYFMRI):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEZDIR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>47</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="btnStable">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Stable</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/icons8-administrative-tools-100.png</normaloff>./icons/icons8-administrative-tools-100.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="btnDev">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Developing</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/icons8-settings-100.png</normaloff>./icons/icons8-settings-100.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Update</string>
    </attribute>
    <widget class="QComboBox" name="cbSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>491</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btnUpdate">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset resource="../icon.qrc">
       <normaloff>./icons/icons8-installing-updates-100.png</normaloff>./icons/icons8-installing-updates-100.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../icon.qrc">./icons/icons8-brain-128.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>160</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>./icons/icons8-shutdown-100.png</normaloff>./icons/icons8-shutdown-100.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAbout">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>160</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact Us</string>
   </property>
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>./icons/icons8-whatsapp-100.png</normaloff>./icons/icons8-whatsapp-100.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
